Tests/UnitTests/Misc/__Snapshots__/AnyEncodableTests/iOS13-testNestedDictionary.1.json

```diff
@@ -7,8 +7,8 @@
       "b22" : "https:\/\/google.com"
     },
     "b3" : [
-      20.199999999999999,
-      19.989999999999998,
+      20.2,
+      19.99,
```

Comment:
This was making watchos tests fail, so I rerun it locally and this snapshot got updated. I am not sure why it's using iOS13 snapshot for watchos tests though https://app.circleci.com/pipelines/github/RevenueCat/purchases-ios/19032/workflows/8cf09d4d-c301-4d1a-b591-f2af92061a78/jobs/204880/steps 

@aboedo do you know?

Reply:
not a clue. might very well be a copy/paste bug